compiler/parser/src/snapshots/rustpython_parser__parser__tests__with_statement.snap

```diff
@@ -761,12 +761,12 @@ expression: "parse_program(source, \"<test>\").unwrap()"
                     context_expr: Located {
                         location: Location {
                             row: 10,
-                            column: 6,
+                            column: 5,
                         },
                         end_location: Some(
                             Location {
                                 row: 10,
-                                column: 7,
+                                column: 8,
```

Comment:
It looks like the location of `(0)` in the following line changed:

https://github.com/RustPython/RustPython/blob/c7faae9b22ce31a3ba1f2cc1cd3ad759b54ce100/compiler/parser/src/parser.rs#L275

Reply:
```
import ast

print(ast.dump(ast.parse("with (0) as x: pass"), include_attributes=True, indent=2))
```

gives:

```
Module(
  body=[
    With(
      items=[
        withitem(
          context_expr=Constant(
            value=0,
            lineno=1,
            col_offset=6,
            end_lineno=1,
            end_col_offset=7),
          optional_vars=Name(
            id='x',
            ctx=Store(),
            lineno=1,
            col_offset=12,
            end_lineno=1,
            end_col_offset=13))],
      body=[
        Pass(
          lineno=1,
          col_offset=15,
          end_lineno=1,
          end_col_offset=19)],
      lineno=1,
      col_offset=0,
      end_lineno=1,
      end_col_offset=19)],
  type_ignores=[])
```

Reply:
The location of `(0)` in the following line didn't change.

https://github.com/RustPython/RustPython/blob/c7faae9b22ce31a3ba1f2cc1cd3ad759b54ce100/compiler/parser/src/parser.rs#L274

Reply:
I've been modifying python.lalrpop, but no luck so far.